dynawo/sources/Models/Modelica/Dynawo/Electrical/Shunts/ShuntB.mo

```diff
@@ -0,0 +1,52 @@
+within Dynawo.Electrical.Shunts;
+
+/*
+* Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+model ShuntB "Shunt element with constant susceptance, reactive power depends on voltage"
+
+  import Dynawo.Connectors;
+  import Dynawo.Types;
+  import Dynawo.Electrical.Controls.Basics.SwitchOff;
+  import Modelica;
+
+  extends SwitchOff.SwitchOffShunt;
```

Comment:
It could be nice also if you could add an Icon as mentioned in the discussion :)

Reply:
Of course. Shall I add the Icon in AdditionalIcons and extend it within the Shunt model oder shall I add the Icon directly in the shunt model in the icon view?

Reply:
You can add the Icon in AdditionalIcons and then extend it. We will use it also for the other shunt model. As we didn't know if you had started to do it we haven't put one yet